src/IAstriaWithdrawer.sol

```diff
@@ -1,7 +1,9 @@
 // SPDX-License-Identifier: MIT or Apache-2.0
 pragma solidity ^0.8.21;
 
-abstract contract IAstriaWithdrawer {
+import {Ownable} from "lib/openzeppelin-contracts/contracts/access/Ownable.sol";
```

Comment:
why is ownable imported? doesn't appear to be used anywhere?

Reply:
`IAstriaWithdrawer` extends it and it's `onlyOwner` modifier is being used here as well